README.md

```diff
@@ -47,7 +47,28 @@ postgres db. For example, you can print out the database metadata:
 docker-compose exec manage python3 print_schema.py
 ```
 
-# Use the python package
+# Backup and restore the db
+Important: The database docker-compose must be up and running (as described above) before running any of the following steps.
```

Comment:
Maybe also mention that migrations need to be applied too.

Reply:
Agreed and done.